ibis/pyspark/tests/test_timecontext.py

```diff
@@ -13,3 +15,172 @@ def test_table_with_timecontext(client):
     expected = table.compile().toPandas()
     expected = expected[expected.time.between(*context)]
     tm.assert_frame_equal(result, expected)
+
+
+def test_window_with_timecontext(client):
+    table = client.table('time_indexed_table')
+    context = (
+        pd.Timestamp('20170102 07:00:00', tz='UTC'),
+        pd.Timestamp('20170103', tz='UTC'),
+    )
+    window1 = ibis.trailing_window(
+        preceding=ibis.interval(hours=1), order_by='time', group_by='key'
+    )
+    window2 = ibis.trailing_window(
```

Comment:
Is forward looking window supported?

Reply:
Yes, it is. I add a param to test forward-looking window.